### PR TITLE
Fix: corrigir mensagens de validação para ShelterRequest e PetRequest

### PR DESCRIPTION
**Pull request by hywenklis**
## Descrição
- Esta pull request corrige as mensagens de validação nas classes ShelterRequest e PetRequest. Foram feitos ajustes nas mensagens em inglês para garantir que estejam semanticamente corretas e claras para o usuário final.

### O que foi feito:
- Correção das mensagens de validação nas classes ShelterRequest e PetRequest.
- As mensagens agora utilizam uma linguagem mais clara e adequada em inglês.

### Por que foi feito:
- As mensagens anteriores possuíam erros gramaticais e de semântica, o que poderia causar confusão para usuários que falam inglês. Esta alteração garante uma melhor experiência de usuário.
- https://github.com/hywenklis/buddy-api/issues/81 

## Tipos de mudanças
- [ ] Correção de bug (mudança que corrige uma issue)
- [ ] Nova funcionalidade (mudança que adiciona uma nova funcionalidade)
- [x] Refatoração (melhoria de código que não altera o comportamento do sistema)
- [ ] Alteração de documentação

## Checklist
- [x] O código segue o padrão de estilo e boas práticas do projeto.
- [x] Eu fiz uma revisão manual do código.
- [x] Testes foram escritos ou ajustados para cobrir as mudanças.
- [x] A documentação foi atualizada, se necessário.

### CODE REVIEW

**Review by genesluna — DISMISSED** [review]

**Comment by hywenklis** [comment]
@CodiumAI-Agent /review
--pr_reviewer.inline_code_comments=true

**Comment by hywenklis** [comment]
@CodiumAI-Agent /review
--pr_reviewer.extra_instructions="
In the 'possible issues' section, emphasize the following:
- Is the code logic efficient in terms of performance and complexity?
- Are there any redundant or unnecessary operations that could be optimized or removed?
- Are there potential scalability issues if the code needs to handle larger datasets or higher traffic?
- Does the code follow SOLID principles (Single Responsibility, Open/Closed, Liskov Substitution, Interface Segregation, Dependency Inversion)?
- Is the code clean and maintainable, adhering to Clean Code practices such as meaningful naming, proper function size, and clear error handling?
- Are there proper abstractions in place to avoid code duplication or hardcoded logic?
- Does the implementation leverage new features and improvements introduced in Java 21 (e.g., pattern matching, virtual threads) and Spring Boot 3+ (e.g., new configuration options, observability)?
- Are unit tests covering all critical paths and edge cases, and are integration tests ensuring the proper functioning of external components (e.g., database, message brokers)?
- Are there any missing test cases, especially for error scenarios or edge cases?
"

**Comment by hywenklis** [comment]
@CodiumAI-Agent /review
--pr_reviewer.extra_instructions="
In the 'possible issues' section, emphasize the following:
- Is the code logic efficient in terms of performance and complexity?
- Are there any redundant or unnecessary operations that could be optimized or removed?
- Are there potential scalability issues if the code needs to handle larger datasets or higher traffic?
- Does the code follow SOLID principles (Single Responsibility, Open/Closed, Liskov Substitution, Interface Segregation, Dependency Inversion)?
- Is the code clean and maintainable, adhering to Clean Code practices such as meaningful naming, proper function size, and clear error handling?
- Are there proper abstractions in place to avoid code duplication or hardcoded logic?
- Does the implementation leverage new features and improvements introduced in Java 21 (e.g., pattern matching, virtual threads) and Spring Boot 3+ (e.g., new configuration options, observability)?
- Are unit tests covering all critical paths and edge cases, and are integration tests ensuring the proper functioning of external components (e.g., database, message brokers)?
- Are there any missing test cases, especially for error scenarios or edge cases?
"
--pr_reviewer.num_code_suggestions="8"

**Comment by hywenklis** [comment]
@CodiumAI-Agent /improve
--pr_code_suggestions.extra_instructions="
Emphasize the following:
- Is the code logic efficient in terms of performance and complexity?
- Are there any redundant or unnecessary operations that could be optimized or removed?
- Are there potential scalability issues if the code needs to handle larger datasets or higher traffic?
- Does the code follow SOLID principles (Single Responsibility, Open/Closed, Liskov Substitution, Interface Segregation, Dependency Inversion)?
- Is the code clean and maintainable, adhering to Clean Code practices such as meaningful naming, proper function size, and clear error handling?
- Are there proper abstractions in place to avoid code duplication or hardcoded logic?
- Does the implementation leverage new features and improvements introduced in Java 21 (e.g., pattern matching, virtual threads) and Spring Boot 3+ (e.g., new configuration options, observability)?
- Are unit tests covering all critical paths and edge cases, and are integration tests ensuring the proper functioning of external components (e.g., database, message brokers)?
- Are there any missing test cases, especially for error scenarios or edge cases?
"

--pr_code_suggestions.num_code_suggestions_per_chunk="10"
--pr_code_suggestions.commitable_code_suggestions=true

**Comment by hywenklis** [comment]
@CodiumAI-Agent /improve
--pr_code_suggestions.extra_instructions="
Emphasize the following:

- Is the code logic efficient in terms of performance and complexity, minimizing time and space overhead?
- Are there any redundant or unnecessary operations that could be refactored, optimized, or removed to enhance maintainability and performance?
- Are there potential scalability issues, especially considering larger datasets, high traffic, or distributed environments?
- Does the code adhere to SOLID principles (Single Responsibility, Open/Closed, Liskov Substitution, Interface Segregation, Dependency Inversion) to promote modularity and flexibility?
- Is the code clean, easy to understand, and maintainable, following Clean Code practices such as clear naming conventions, proper function and class sizing, and appropriate error handling?
- Are there well-structured abstractions to avoid code duplication, hardcoded values, and unnecessary complexity?
- Does the implementation effectively utilize new features in Java 21 (e.g., pattern matching, virtual threads) and Spring Boot 3+ (e.g., enhanced configuration, observability) to improve efficiency and clarity?
- Are unit tests covering all critical logic paths, including edge cases, and are integration tests properly verifying the functionality of external components such as databases and message brokers?
- Are there any missing test cases for edge scenarios, error handling, or unexpected input?
- Do all test cases align with the defined engineering guidelines, including appropriate levels of coverage for both unit and integration tests?
"
--pr_code_suggestions.num_code_suggestions_per_chunk="10"
--pr_code_suggestions.commitable_code_suggestions=true
--pr_code_suggestions.follow_engineering_guidelines="https://www.notion.so/buddydocs/Diretrizes-de-Engenharia-faf7c184fb784405a2b7b36f4da72976"

File: src/main/java/com/buddy/api/web/pets/requests/PetRequest.java [hunk 1]
```diff
@@ -9,36 +9,35 @@
...
+                         @NotBlank(message = "Pet name is mandatory")
+                         String name,
 
-    @Schema(description = "Species of the pet",
-        example = "Cachorro, Gato, Pássaro, Peixe, Réptil")
-    @NotBlank(message = "Specie of mandatory pet")
-    String specie,
+                         @Schema(description = "Species of the pet",
+                                 example = "Cachorro, Gato, Pássaro, Peixe, Réptil")
+                         @NotBlank(message = "Pet species is mandatory")
```
[the remark this answers is not in the document]
Comment (FSLucas22) [reply src/main/java/com/buddy/api/web/pets/requests/PetRequest.java:18]: Entendo que não deve ser restritiva a estes tipos e que a implementação atual está correta.

**Review by FSLucas22 — APPROVED** [review]
